src/request-data/resources.js

```diff
@@ -15,18 +15,21 @@ import { mergeDeepLeft } from 'ramda';
 import configDefaults from '../config';
 import { computeIfExists, hasVerbs, setupOption, transformForm } from './util';
 import { noargs } from '../util/util';
+import { _container } from './resource';
+import UserPreferences from './user-preferences/preferences';
 
 export default ({ i18n }, createResource) => {
   // Resources related to the session
   createResource('session');
```

Comment:
If you assign this resource to a variable, you can reference it below.

```suggestion
  const session = createResource('session');
```